test/pom.xml

```diff
@@ -157,7 +156,7 @@
 							<targetPath>testtool</targetPath>
 						</resource>
 						<webResource>
-							<directory>src/main/webapp/META-INF</directory>
+							<directory>src/main/webapp/META-INF-unfiltered</directory>
```

Comment:
Ah, didn't know Tomcat in Eclipse will break on this.

The benefit of running war:inplace is that is saves some time (no need to create war file) and you can edit html (and related files) in webapp and see the changes immediately without the need to rebuild the war file an restart Tomcat. But for iaf-test this is not really an issue as it gets all the html from the webapp module so when you need to edit html it's better to run iaf-webapp.

Reply:
I reverted META-INF-unfiltered and added a property filteringMetaInf. I think element filteringDeploymentDescriptors can be removed (no ${*} in web.xml and couldn't find a reason in the commit that added it) but in case it is needed for something I can also add a property filteringDeploymentDescriptors.

P.s did an attempt on building the war and using it but gives considerable overhead and needs a lot more changes to the build.xml that can already run Frank! projects with maven war:inplace (which gives the Frank! developer the possibility to edit html without rebuilding and restarting).